src/drivers/tty/serial/ttys_processing.c

```diff
@@ -41,23 +41,30 @@ static inline struct uart *tty2uart(struct tty *tty) {
 	return tu->uart;
 }
 
-static int uart_rx_buff_put(struct uart *dev, int c) {
+static int uart_rx_buff_put(unsigned int irq_nr, struct uart *dev) {
```

Comment:
Is 'irq_nr' used here?

Reply:
No! actually I kept it as it was in `uart_irq_handler`. I'll remove it.